packages/devtools_app/lib/src/flutter/utils.dart

```diff
@@ -14,3 +15,26 @@ Future<void> launchUrl(String url, BuildContext context) async {
     Notifications.of(context).push('Unable to open $url.');
   }
 }
+
+List<TextSpan> maybeConvertToAnsiText(String input, TextStyle defaultStyle) {
```

Comment:
nit: I wouldn't use the word maybe in this method name.
maybe name it
`processAnsiTerminalCodes` instead
The reason I recommend omitting the word maybe as a List<TextSpan> is generated regardless even if the text doesn't happen to have ansi terminal codes.